happi/cli.py

```diff
@@ -982,6 +985,13 @@ def audit(
         print(check_pt)
         return
 
+    if details:
+        check_fns = [fn for fn in checks if details in fn.__name__]
+        for fn in check_fns:
+            click.echo(inspect.getsource(fn))
+
+        return
```

Comment:
What makes one of these like `--details` return early but some others not? The idea being that we should only do `--audit --details` and then call `happi audit --check`?

Reply:
yep, the `--details` flag is meant to be similar to `--list` , to be used more as a diagnostic or in preparation before the audit is run.